src/components/PrivacyStatement/index.js

```diff
@@ -0,0 +1,55 @@
+import { h, Component} from 'preact'
+
+import theme from '../Theme/style.css'
+import style from './style.css'
+import Title from '../Title'
+import {preventDefaultOnClick} from '../utils'
+import {sendScreen} from '../../Tracker'
+import {parseI18nWithXmlTags} from '../../locales'
+
+const externalUrls = {
+  terms: process.env.ONFIDO_TERMS_URL,
+  privacy: process.env.ONFIDO_PRIVACY_URL
+}
+
+class PrivacyStatement extends Component {
+  componentDidMount() {
+    sendScreen(['privacy'])
```

Comment:
can't just use the tracking component wrapper for this?

Reply:
We can but the screen name on woopra will be appended to the capture component one, resulting in something like `screen_document_front_capture_privacy` and we just want `screen_privacy`

Reply:
I wonder if we should have as an option to get rid of the prefixes